### PR TITLE
Beam2DWithStochasticMaterial

### PR DESCRIPTION
**Pull request by odyred**

### CODE REVIEW

File: ISAAR.MSolve.SamplesConsole/Program.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ private static void SolveBuildingInNoSoilSmall()
         {
             VectorExtensions.AssignTotalAffinityCount();
             Model model = new Model();
+            Model model_1 = new Model();
```
Comment (georgestavroulakis) [inline ISAAR.MSolve.SamplesConsole/Program.cs]: No need for construction of the model_1 object. Please explain...

File: ISAAR.MSolve.Analyzers/ISAAR.MSolve.Analyzers.csproj [hunk 2]
```diff
@@ -77,8 +77,8 @@
     <ErrorReport>prompt</ErrorReport>
```
Comment (georgestavroulakis) [inline ISAAR.MSolve.Analyzers/ISAAR.MSolve.Analyzers.csproj:77]: Remove csproj commits if there is no new file addition

File: ISAAR.MSolve.Analyzers/packages.config [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (georgestavroulakis) [inline ISAAR.MSolve.Analyzers/packages.config:1]: Remove packages.config commit if there is no new package

**Review by GoatOfRafin — CHANGES_REQUESTED** [review]
Please check comments and resubmit at development branch

**Pull request closed without merging**